src/components/user/ChangePassword.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import lockIcon from '../../assets/lock-icon.png'; // Path to lock icon asset
+import logoWithCircleBorder from '../../assets/logo_with_circle_border-removebg.png'; // Path to logo asset
+
+// Styles
+const pageStyles = () => ({
+  width: '100%',
+  minHeight: '100vh',
+  overflow: 'auto',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+});
+
+const backgroundStyles = () => ({
+  backgroundColor: 'rgba(91, 46, 72, 1)',
```

Comment:
we have to be careful adding these infile color changes, maybe we can make this one of the tailwind config colors so it can be used globally?

Reply:
I agree. What is that color? On my branch, I made custom colors for our website theme in tailwind.config.js.